servers.json

```diff
@@ -788,5 +788,22 @@
         "addresses": [
             "originrealms.com"
         ]
+    },
+    {
+        "id": "fyre",
+        "name": "Fyre",
+        "addresses": [
+            "fyre.cc",
+            "play.fyre.cc"
```

Comment:
Please take a look at:
[the ip-address section in the README](https://github.com/LunarClient/ServerMappings/blob/master/README.md#ip-addresses)

You do not need to add sub-domains to the addresses array.